src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'https:fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap)';
```

Comment:
There is a syntax error in the import statement for the Google Fonts. The URL is missing an opening parenthesis after 'https:'. It should be '@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');'.